cypress/e2e/Admin/AdminView.cy.js

```diff
@@ -5,16 +5,18 @@ import {
 import { CONTEXTS } from '../../../src/config/contexts';
 import {
   ADD_NEW_QUESTION_TITLE_CY,
+  ANALYTICS_CONTAINER_CY,
   CREATE_VIEW_CONTAINER_CY,
+  NAVIGATION_ANALYTICS_BUTTON_CY,
   NAVIGATION_CREATE_QUIZ_BUTTON_CY,
   NAVIGATION_RESULT_BUTTON_CY,
   NAVIGATION_TAB_CONTAINER_CY,
   TABLE_BY_QUESTION_CONTAINER_CY,
   buildTableByQuestionCy,
   dataCyWrapper,
 } from '../../../src/config/selectors';
-import { APP_DATA } from '../../fixtures/appData';
-import { APP_SETTINGS } from '../../fixtures/appSettings';
+import { APP_DATA_2 } from '../../fixtures/appData';
+import { APP_SETTINGS_2 } from '../../fixtures/appSettings';
```

Comment:
Can we give better names?

Reply:
I renamed them to try to give a little more info about their differences.

Let me know If you had other name ideas